heat/fft/fft.py

```diff
@@ -0,0 +1,787 @@
+        along the axis specified by `axis`. If `n` is smaller than the length of the input, the input is cropped. If `n` is
+        larger, the input is padded with zeros. Default: None.
+    axis : int, optional
+        Axis over which to compute the inverse FFT. If not given, the last axis is used, or the only axis if x has only one dimension. Default: -1.
+    norm : str, optional
+        Normalization mode: 'forward', 'backward', or 'ortho' (see `numpy.fft` for details). Default is "backward".
+
+    Notes
+    -----
+    This function requires MPI communication if the input array is transformed along the distribution axis.
+    If the input array is 1-D and distributed, this function copies the entire array on each MPI process! i.e. if the array is very large, you might run out of memory.
+    Hint: if you are looping through a batch of 1-D arrays to transform them, consider stacking them into a 2-D DNDarray and transforming them all at once (see :func:`ifft2`).
+    """
+    return __fft_op(x, torch.fft.ifft, n=n, axis=axis, norm=norm)
+
+
+def ifft2(
+    x: DNDarray, s: Tuple[int, int] = None, axes: Tuple[int, int] = (-2, -1), norm: str = None
+) -> DNDarray:
+    """
+    Compute the 2-dimensional inverse discrete Fourier Transform.
+
+    Parameters
+    ----------
+    x : DNDarray
+        Input array, can be complex
+    s : Tuple[int, int], optional
+        Shape of the output along the transformed axes. (default is x.shape)
+    axes : Tuple[int, int], optional
+        Axes over which to compute the inverse FFT. If not given, the last `len(s)` axes are used, or all axes if `s` is
+        also not specified. Repeated indices in `axes` means that the transform over that axis is performed multiple
+        times. (default is (-2, -1))
+    norm : str, optional
+        Normalization mode: 'forward', 'backward', or 'ortho' (see `numpy.fft` for details). Default is "backward".
+
+    Notes
+    -----
+    This function requires MPI communication if the input array is distributed and the split axis is transformed.
+    """
+    return __fftn_op(x, torch.fft.ifft2, s=s, axes=axes, norm=norm)
+
+
+def ifftn(
+    x: DNDarray, s: Tuple[int, int] = None, axes: Tuple[int, ...] = None, norm: str = None
+) -> DNDarray:
+    """
+    Compute the N-dimensional inverse discrete Fourier Transform.
+
+    Parameters
+    ----------
+    x : DNDarray
+        Input array, can be complex
+    s : Tuple[int, ...], optional
+        Shape of the output along the transformed axes. (default is x.shape)
+    axes : Tuple[int, ...], optional
+        Axes over which to compute the inverse FFT. If not given, the last `len(s)` axes are used, or all axes if `s` is
+        also not specified. Repeated indices in `axes` means that the transform over that axis is performed multiple
+        times. (default is None)
+    norm : str, optional
+        Normalization mode: 'forward', 'backward', or 'ortho' (see `numpy.fft` for details). Default is "backward".
```

Comment:
One could also add a reference to the PyTorch-docs here, because they use the same options for `norm`

Reply:
I've removed the pointer to [numpy.fft](https://numpy.org/doc/stable/reference/routines.fft.html) for now, however that page does have a lot more infos than the torch.fft page. I think it would be good to mention the numpy page in the docs because it's so exhaustive.